src/database/datatools/constructions.py

```
@@ -0,0 +1,209 @@
+                        layer15: Optional[str] = None,
+                        layer16: Optional[str] = None,
+                        layer17: Optional[str] = None,
+                        layer18: Optional[str] = None,
+                        layer19: Optional[str] = None,
+                        layer20: Optional[str] = None) -> None:
+    
+    conn = sqlite3.connect(db_path)
+    conn.row_factory = sqlite3.Row
+    cursor = conn.cursor()
+    def fetch_id(mat_name):
+        if mat_name is None: return None
```

Ruff (0.14.14)

[error] Multiple statements on one line (colon)

(E701)